qa/workunits/ceph-helpers.sh

```diff
@@ -293,7 +293,7 @@ function test_kill_daemons() {
     #
     # kill the mon and verify it cannot be reached
     #
-    kill_daemons $dir TERM || return 1
+    kill_daemons $dir TERM mon || return 1
```

Comment:
`kill_daemons` does not accept a forth parameter, see
```sh
# @param pid the process id to send a signal
# @param send_signal the signal to send
# @param delays sequence of sleep times before failure
```

or you need to filter the output of `find $dir -name "*.pid" 2>/dev/null` with a prefix of "mon" so it only kills `mon.*pid` under the testdir?